File: 백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

const limit = parseInt(input[0]);
const speed = parseInt(input[1]);
let fee = 0;

if (speed <= limit) {
  fee = 0;
} else if (speed - limit <= 20) {
  fee = 100;
} else if (speed - limit <= 30) {
  fee = 270;
} else {
  fee = 500;
}

console.log(
  fee === 0
    ? 'Congratulations, you are within the speed limit!'
    : `You are speeding and your fine is $${fee}.`,
);

File: 백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.py
limit = int(input())
speed = int(input())
fee = 0
if speed <= limit:
    fee = 0
elif speed - limit <= 20:
    fee = 100
elif speed - limit <= 30:
    fee = 270
else:
    fee = 500

print('Congratulations, you are within the speed limit!' if fee == 0 else f'You are speeding and your fine is ${fee}.')